azure-application-insights-spring-boot-starter/README.md

```diff
@@ -133,16 +133,16 @@ azure.application-insights.logger.level=error
 azure.application-insights.channel.in-process.developer-mode=false
 # Endpoint address, Default value: https://dc.services.visualstudio.com/v2/track.
 azure.application-insights.channel.in-process.endpoint-address=https://dc.services.visualstudio.com/v2/track
-# Maximum count of telemetries that will be batched before sending. Default value: 500.
+# Maximum count of telemetries that will be batched before sending. Must be between 1 and 1000. Default value: 500.
 azure.application-insights.channel.in-process.max-telemetry-buffer-capacity=500
-# Interval to send telemetry. Default value: 5 seconds.
+# Interval to send telemetry. Must be between 1 and 300. Default value: 5 seconds.
 azure.application-insights.channel.in-process.flush-interval-in-seconds=5
-# Size of disk that we can use. Default value: 10 megabytes.
+# Size of disk that we can use. Must be between 1 and 1000. Default value: 10 megabytes.
 azure.application-insights.channel.in-process.max-transmission-storage-files-capacity-in-mb=10
 # Enable/Disable throttling on sending telemetry data. Default value: true.
 azure.application-insights.channel.in-process.throttling=true
 
-# Percent of telemetry events that will be sent to Application Insights. Default value: 100% (all telemetry events).
+# Percent of telemetry events that will be sent to Application Insights. Must be between 0.0 and 100.0. Default value: 100 (all telemetry events).
```

Comment:
We should add this comment here : 
"All sampling percentage must be in a ratio of 100/N where N is a whole number (2, 3, 4, …). E.g. 50 for 1/2 or 33.33 for 1/3.
     *  Failure to follow this pattern can result in unexpected / incorrect computation of values in the portal." 

This is important. We can only set specific sampling percentages for UI to be consistent

Reply:
I didn't understand, can it be 2/3?

Reply:
so 100/N = Sampling ratio. 
Eg 100/2  = 50
100/3 = 33
100/4 = 25
100/5 = 20
100/10 = 10 etc.

Reply:
probably we can create property for N instead? It would be less error prone if we say that N must be between 1 (100%) and 100 (1%).

what about 0%, is it valid?

Reply:
just read the documentation for .NET, it can be 1000 (0.1%) as well. Looks really odd to me, I mean that 1/3 is not really 33 or 33.3 or 33.33, it's 33.(3) which you can properly specify in properties and for this reason per documentation I have to chose closest percentage to 100/N.